test/units/cli/test_vault.py

```diff
@@ -142,6 +141,34 @@ def test_encrypt_string_stdin(self, mock_stdin_read, mock_vault_editor, mock_set
         cli.parse()
         cli.run()
 
+    @patch('ansible.cli.vault.VaultCLI.setup_vault_secrets')
+    @patch('ansible.cli.vault.VaultEditor')
+    def test_encrypt_string_stdin_prompt_error(self, mock_vault_editor, mock_setup_vault_secrets):
+        mock_setup_vault_secrets.return_value = [('default', TextVaultSecret('password'))]
+        cli = VaultCLI(args=['ansible-vault',
+                             'encrypt_string',
+                             '--stdin-name',
+                             'the_var_from_stdin',
+                             '--prompt'])
+
+        with pytest.raises(SystemExit):
```

Comment:
Please add a `match=` argument so that it's documented what's expected.

Reply:
I did try to do this originally, but found that no matter what I did the message of the error would always simply be "2", so it's not possible to match in that way. 

The message is printed to stderr, so usually I would have suggested using the capsys fixture to capture the message, however because the mock decorators are used on this function I can't get that to work either.

I'm probably missing something obvious, apologies for the trouble!